### DISCUSSION TITLE
SkyFactory 4

### DISCUSSION THREAD

**Discussion by FabMaster2000** [itzg/docker-minecraft-server#3207] [Q&A]
Hello all,

today i started an minecraft server with the SkyFactory 4 mod. Installed quickly and easy and after I logged in at curse forge for the client application. But when i will login to the server it says, that there are wrong versions of a few mod packages. 

So, how is that possible and how can I fix it?

Thank

**Comment by itzg** [comment]
Perhaps double check that the same version of the modpack was selected on both ends.
